src/main/types/applicabilityScanResults.ts

```diff
@@ -1,21 +1,16 @@
-/*************************************************************
- * The following logic is part of the CVE applicability scan.*
- * It will be hidden until it is officially released.        *
- * ***********************************************************
- */
-// export interface ApplicabilityScanResults {
-//     results: Map<string, ApplicabilityScanResult[]>;
-//     scanners_ran: string[];
-// }
+export interface ApplicabilityScanResults {
+    results: Map<string, ApplicabilityScanResult[]>;
+    scanners_ran: string[];
```

Comment:
What is scanners_ran? Please explain in a comment.

Reply:
All CVEs that the Applicability scanner searched for (regardless of whether they were found)